building-and-research/human-in-the-loop.md

```diff
@@ -0,0 +1,45 @@
+---
+nav_order: tbd
+parent: Building and research
+title: Human-in-the-loop
+description: Machine translation supported by human post-editing or evaluation
+---
+
+## Definition
```

Comment:
We can omit this title as we have not used it in other articles.
The front matter `title` will be inserted as `<h1>`, no there's so need for a subtitle.